bib/internal/container/container_test.go

```diff
@@ -171,3 +175,52 @@ echo '%s'
 		assert.ErrorContains(t, err, "unsupported root filesystem type: ext1, supported: ")
 	}
 }
+
+func subscribeMachine(t *testing.T) (restore func()) {
+	if _, err := exec.LookPath("subscription-manager"); err != nil {
+		t.Skip("no subscription-manager found")
+		return func() {}
+	}
+
+	matches, err := filepath.Glob("/etc/pki/entitlement/*.pem")
+	if err == nil && len(matches) > 0 {
```

Comment:
Yeah, it's basically just double checking if the machine is subscribed already (because the helper unsubscribes machines that got subscribed just for the test again), actually there should be a comment to explain this and a "XXX: find a better way", I had trouble finding some machine readable API and stuff like "subscription-manager status"/"list" is both slow and hard to parse. But maybe I'm missing something :/